base/latexrelease.dtx

```diff
@@ -746,8 +746,12 @@ of this package available from CTAN}
 %
 % Finally, redirect the error thrown by \cs{NewHook} to nowhere so it
 % can be safely reused (the hook isn't redeclared if it already exists).
+% The same happens for \cs{NewMarkClass}.  
+% \changes{v1.0p}{2022/11/14}
+%         {Silence \cs{NewMarkClass} in rollback (gh/955)}
 %    \begin{macrocode}
 \msg_redirect_name:nnn { hooks } { exists } { none }
+\msg_redirect_name:nnn { mark } { class-already-defined }{ none } 
```

Comment:
ah, do we actually go through the trouble? well why not, but  my guess was it isn't necessary, because you reprocess a document using LaTeX release only if it did work without in at that time, don't you? In which case it should not have a problem with doing new marks etc with the same name, because if it would have had this problem in the past too.  But I guess it could be an issue if you do a roll forward, and as you say it is easy enough to restore in the end.

Reply:
I prefer restoring because then you always have a predictable state of things (in theory, that is).

Reply:
yes agreed (and it is simple enough)